docs/generate_securedrop_application_key.rst

```diff
@@ -16,14 +16,7 @@ Ensure Filenames are Preserved
 In order to preserve filenames when you decrypt submissions, on each *Secure
 Viewing Station*, you should open a **Terminal** and type the following commands:
 
-.. code:: sh
-
-  cd /live/persistence/TailsData_unlocked/dotfiles
-  cp ~/.bashrc .
-  echo "/usr/bin/dconf write /org/gnome/nautilus/preferences/automatic-decompression false" >> .bashrc
-
-.. note:: This only needs to be done once on each *Secure Viewing Station*.
-          After a reboot it will persist.
+.. include:: includes/tails-svs-nautilus.txt
```

Comment:
I'm curious about why this is .txt instead of .rst here ?

Reply:
When using `include` to minimize duplicated content I followed the suggestion of [this blog](http://tychoish.com/post/large-sphinx-deployments/) which recommended using another extension to make it crystal clear which files are text snippets